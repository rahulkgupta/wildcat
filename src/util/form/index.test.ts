import Form from '@src/util/form';

test('this is a test', () => {
  expect(2).toBe(2);
});
