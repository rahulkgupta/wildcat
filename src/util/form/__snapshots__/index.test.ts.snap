// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form functions getFields 1`] = `
Array [
  Object {
    "id": "ssaf",
    "value": "value",
  },
  Object {
    "id": "submit",
    "value": "value",
  },
]
`;

exports[`Form functions toJSON 1`] = `"{\\"id\\":\\"1234\\",\\"fields\\":[{\\"id\\":\\"ssaf\\",\\"value\\":\\"value\\"},{\\"id\\":\\"submit\\",\\"value\\":\\"value\\"}]}"`;
